Tests/govuk_ios/govuk_ios_unit_tests/Specs/AppConfig/AppConfigServiceTests.swift

```diff
@@ -0,0 +1,55 @@
+@testable import govuk_ios
+import XCTest
+
+final class AppConfigServiceTests: XCTestCase {
+    
+    func test_isFeatureEnabled_whenFeatureFlagIsSetToAvilable_returnsTrue() throws {
+        //Given
```

Comment:
Is this a new pattern we should start applying to our specs (given / when / then)? Feels like it could be helpful but then we probably want to be consistent across the test suite

Reply:
So actually think we should all be using this spec, If you have a look at the onboarding module, its also what has been used

Reply:
Discussed offline and we'll have a follow up chat around testing patterns so we're consistent across the app / libraries but not a blocker for the PR